github/Requester.py

```diff
@@ -540,7 +540,7 @@ def graphql_query(self, query: str, variables: Dict[str, Any]) -> Tuple[Dict[str
         """
         input_ = {"query": query, "variables": {"input": variables}}
 
-        response_headers, data = self.requestJsonAndCheck("POST", "https://api.github.com/graphql", input=input_)
+        response_headers, data = self.requestJsonAndCheck("POST", f"https://{self.__hostname}:{self.__port}/graphql", input=input_)
```

Comment:
Right, that is easier than I thought, but that should do it.

I'd suggest to construct the GraphQl URL in the `__init__` method right after `o = urllib.parse.urlparse(base_url)` (line 397) like this:
```
o = urllib.parse.urlparse(base_url)
self.__graphql_url = f"{o.scheme}://{o.hostname}:{o.port}/graphql"
```

or maybe safer would be this
```
o = urllib.parse.urlparse(base_url)
self.__graphql_url = urlunparse(o._replace(path="graphql"))
```

And then use `self.__graphql_url` here instead:
```suggestion
        response_headers, data = self.requestJsonAndCheck("POST", self.__graphql_url, input=input_)
```

Reply:
Above suggestion using `urlunparse` should also fix the test error `ValueError: Port could not be cast to integer value as 'None'`.